mqtt/mqtt-edgehub/tests/command.rs

```diff
@@ -50,7 +50,7 @@ async fn disconnect_client() {
 
     let topic = "$edgehub/disconnect";
     edgehub_client
-        .publish_qos1(topic, "test-client", false)
+        .publish_qos1(topic, "\"test-client\"", false)
```

Comment:
why? also you can use raw string literals https://doc.rust-lang.org/stable/rust-by-example/std/str.html#literals-and-escapes

Reply:
It needs to be valid json. I can change it to a literal.